control_protocol.md

```diff
@@ -330,17 +330,490 @@ A Component MUST also offer a list of all possibly callable methods in accordanc
+            "name": "set_nodes",
+            "params": [
+                {
+                    "name": "nodes",
+                    "schema": {
+                        "type": "object",
+                        "additionalProperties": true
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "set_remote_components",
+            "params": [
+                {
+                    "name": "components",
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "compose_global_directory",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "object",
+                    "additionalProperties": true
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "compose_local_directory",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "object",
+                    "additionalProperties": true
+                },
+                "required": true
+            },
+            "description": "Compose a dictionary with the local directory."
+        },
+        {
+            "name": "get_component_names",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "clean_addresses",
+            "params": [
+                {
+                    "name": "expiration_time",
+                    "schema": {
+                        "type": "number"
+                    },
+                    "required": true
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            },
+            "description": "Clean all expired addresses from the directory.\n\n        :param float expiration_time: Expiration limit in s.\n        "
```

Comment:
Can we express the unit of the parameter in the schema?

Reply:
I just had a look at the [specification ](https://spec.open-rpc.org/#content-descriptor-object)(I'm not an openrcp expert yet):

Yes, for example (summary and description are two available fields for every parameter/result):
~~~json
{
            "name": "get_polling_interval",
            "params": [],
            "result": {
                "name": "result",
                "schema": {
                    "type": "number"
                },
                "summary": "in ms",
                "description": "whatever",
                "required": true
            }
        },
~~~

Reply:
This seems like a workaround (`summary` seems be intended like the first line of a docstring, `description` everything), but would be possible. 
Maybe just keep including the unit in the description, probably easiest for now?